data/config/default/mermaid.json

```diff
@@ -0,0 +1,6 @@
+{
+  "library": {
+    "path": "https://cdnjs.cloudflare.com/ajax/libs/mermaid/8.0.0/mermaid.min.js"
```

Comment:
Place use unpkg source like the others: `//unpkg.com/mermaid/dist/mermaid.min.js`

Just a personal preference. If you want to use a different source, you will be able to set that in custom config file.